src/SFA.DAS.ApplyService.Application/Apply/Financial/RecordGradeRequest.cs

```diff
@@ -0,0 +1,19 @@
+﻿using MediatR;
+using SFA.DAS.ApplyService.Domain.Entities;
+using System;
+
+namespace SFA.DAS.ApplyService.Application.Apply.Financial
+{
+    public class RecordGradeRequest : IRequest<bool>
+    {
+        public Guid ApplicationId { get; set; }
+        public FinancialReviewDetails FinancialReviewDetails { get; set; }
+        public string FinancialReviewStatus { get; set; }
```

Comment:
As mentioned in the other PR. Would expect this to be calculated within the Handler

Reply:
Done